deps/Makefile

```diff
@@ -591,8 +640,11 @@ update-llvm:
 	([ -d "llvm-$(LLVM_VER)/tools/lldb" ] || exit 0;            cd llvm-$(LLVM_VER)/tools/lldb; git pull --ff-only)
 
 ## LIBUV ##
+LIBUV_GIT_URL=git://github.com/JuliaLang/libuv.git
+LIBUV_TAR_URL=https://github.com/JuliaLang/libuv/tarball/$1
+$(eval $(call git-external,libuv,LIBUV,configure,.libs/libuv.a))
```

Comment:
Possibly. `libuv.a` never gets created by MSVC, but my hacky shell script manually copies `usr/lib/uv.lib` to `usr/lib/libuv.a`. I think `.la` should be safe and a bit more portable, as long as we assume we're using libtool to build libuv. Some day libuv might switch to cmake, they'd probably have to give up on gyp first.